runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -120,6 +138,20 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
     @Override
     public <R> R access(ICorfuSMRAccess<R, T> accessMethod,
                         Object[] conflictObject) {
+        R value;
+        boolean isEnabled = MetricsUtils.isMetricsCollectionEnabled();
```

Comment:
Oops, the above described a consistency example on the server side.  You're pointing to CorfuCompileProxy, sorry.  There's a case there in TXExecuteInner where either counters should either be triggered or not during the loop.